src/server/search/search_family.cc

```diff
@@ -272,6 +352,9 @@ void SearchFamily::Register(CommandRegistry* registry) {
   using CI = CommandId;
 
   *registry << CI{"FT.CREATE", CO::GLOBAL_TRANS, -2, 0, 0, 0}.HFUNC(FtCreate)
+            << CI{"FT.DROPINDEX", CO::GLOBAL_TRANS, -2, 0, 0, 0}.HFUNC(FtDropIndex)
+            << CI{"FT.INFO", CO::GLOBAL_TRANS, 2, 0, 0, 0}.HFUNC(FtInfo)
+            << CI{"FT._LIST", CO::GLOBAL_TRANS, 1, 0, 0, 0}.HFUNC(FtList)
```

Comment:
Why the underscore? Is it because something is missing? Maybe add a comment saying what.

Reply:
Because its the same in Redis https://redis.io/commands/ft._list/

Its "temporary" for a long time already

Reply:
:facepalm: